packages/builder/src/components/automation/AutomationPanel/AutomationPanel.svelte

```diff
@@ -82,6 +101,17 @@
 </Modal>
 
 <style>
+  .nav-group {
+    padding-top: var(--spacing-l);
+  }
+  .nav-group-header {
+    color: var(--spectrum-global-color-gray-600);
+    padding: 0px calc(var(--spacing-l) + 4px);
+    padding-bottom: var(--spacing-l);
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
```

Comment:
Great future-proofing here, considering I don't think any of the names even can overflow at the mo 🫡